### PR TITLE
With subdir

### PR DESCRIPTION
**Pull request by orangewolf**

### CODE REVIEW

**Comment by alishaevn** [comment]
looks good to me, but will let someone else approve.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -2,9 +2,14 @@ name: "Build Docker Images"
 on:
   workflow_dispatch:
     inputs:
+      image_name:
```
Comment (aprilrieger) [inline .github/workflows/build.yaml:5]: Can you add description to these? Or reply with them and I can add them?

Comment (aprilrieger) [reply .github/workflows/build.yaml:5]: helm image, repository image?

Comment (aprilrieger) [reply .github/workflows/build.yaml:5]: Thanks Rob!

**Review by labradford — APPROVED** [review]

**Pull request merged**